boot/zephyr/boards/nrf52840_pca10056_usb_cdc_log.conf

```diff
@@ -0,0 +1,37 @@
+# The UART is used for Serial Recovery, so logging requires
```

Comment:
Is there any way to do this conditionally with if statements in the Kconfig file instead of using a board-specific file? This makes it not portable to other boards.

Reply:
ok, will try to do that